music/src/music_rnn/model.ts

```diff
@@ -57,20 +60,24 @@ export class MusicRNN<A extends controls.ControlSignalUserArgs> {
    * converting between `NoteSequence` and `Tensor` objects. If not provided, a
    * `converter.json` file must exist within the checkpoint directory specifying
    * the type and args for the correct `DataConverter`.
-   * @param controlSignal (Optional) A `ControlSignal` object that produces
-   * control tensors that will be appended to model inputs.
+   * @param chordEncoder (Optional) A `ChordEncoder` object that converts chord
+   * symbol strings to model input tensors.
```

Comment:
Mention that these are concatenated before(after?) the input tensor from the data converter.

Reply:
Done.